### ISSUE TITLE
Style React-Native element

### ISSUE THREAD

**Issue by Gerharddc** [headzoo/react-moment#55]
I see that I am supposed to provide a class type like 'Text' to the element prop in React-Native. This works but I cannot figure out how to style that element because I am can just pass the class as a type and Moment does not have a 'style' prop for forwarding. How should I do this?

**Comment by nicovanzyl** [comment]
+1 on this issue

**Comment by ajboxjr** [comment]
Yes it is possible to add a styling prop to the Moment component. Check it out!
```
<Moment style={this.props.style} tz={DeviceInfo.getTimezone()} element={Text} format="MMMM Do hh:mmA">
            {this.props.date}
      </Moment>
```

**Comment by headzoo** [comment]
Closing old open issue during cleanup.
